src/interactive/Dropdown.jsx

```diff
@@ -72,11 +106,15 @@ class Dropdown extends React.PureComponent {
 	componentDidMount() {
 		document.body.addEventListener('click', this.onBodyClick);
 		document.body.addEventListener('keydown', this.onBodyKeyDown);
+		window.addEventListener('resize', this.getContentPosition);
+		document.addEventListener('scroll', this.getContentPosition, true);
```

Comment:
Might be good to figure out how to do this only when the dropdown is opened

Reply:
The listener is fine to leave in place while the component is mounted, but I'd recommend throttling, debouncing, or `requestAnimationFrame` on the `resize` and `scroll` handler functions